src/theory/uf/equality_engine.h

```diff
@@ -767,6 +782,22 @@ class EqualityEngine : public context::ContextNotifyObj {
   void explainPredicate(TNode p, bool polarity, std::vector<TNode>& assertions,
                         EqProof* eqp = nullptr) const;
 
+  //--------------------------- standard safe explanation methods
+  /**
+   * Explain literal, add its explanation to assumptions. This method does not
+   * add duplicates to assumptions. It additionally contains assertions for
+   * whether literal does hold in this class. If lit is a disequality, it
```

Comment:
```suggestion
   * add duplicates to assumptions. It requires that the literal
   * holds in this class. If lit is a disequality, it
```

(I was confused about "assertions" referring to C++ assertions here.